appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: build{build}
+
+
+environment:
+  GherkinVersion: "3.3.0"
+  GherkinPackageVersion: "$(GherkinVersion)-ci$(APPVEYOR_BUILD_NUMBER)"
+
+
+nuget:
+  project_feed: true
+  disable_publish_on_pr: true
+
+configuration: Release
+
+
+before_build:
+  - appveyor DownloadFile https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
+  - appveyor DownloadFile https://raw.githubusercontent.com/appveyor/ci/master/scripts/nuget-restore.cmd
+  - nuget-restore dotnet\Gherkin.CSharp.sln
```

Comment:
I think it would make more sense to move this entire file to the `dotnet` directory, and have Appveyor build the [gherkin-dotnet](https://github.com/cucumber/gherkin-dotnet) project instead. (We regularly `git subtree push` individual modules to independent repositories).

WDYT?